apps/builder/app/routes/auth.ws.ts

```diff
@@ -0,0 +1,37 @@
+import type { LoaderFunction } from "@remix-run/node";
+import createDebug from "debug";
+import { builderAuthenticator } from "~/services/builder-auth.server";
+import { isBuilderUrl } from "~/shared/router-utils/origins";
+
+const debug = createDebug("Auth").extend("Ws");
```

Comment:
we need to decide on convention we use for naming, I would prefer using exact module name
```suggestion
const debug = createDebug("auth").extend("ws");
```

Reply:
why do we extend"ws" here  but not in other places?